modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -11,7 +11,9 @@
 
 This section describes how to calculate resources, such as memory and CPU, required to run {prod}.
 
-Both the {prod-short} central controller and user workspaces consist of a set of containers. Those containers contribute to the resources consumption in terms of CPU and RAM limits and requests.
+{prod-short} operator, {devworkspace} controller, and user workspaces consist of a set of pods. Those pods contribute to the resources consumption in terms of CPU and RAM limits and requests.
```

Comment:
```suggestion
The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods. Those pods contribute to the resource consumption in terms of CPU and RAM limits and requests.
```

Reply:
In this PR, operator and controller are not consistently capitalized. Judging from the rest of the content, it seems these words should be capitalized. However, if that assumption is wrong, all should be moved to lower case first letters.